releasenotes/notes/0.12/fix-ndarray-contiguity-e903d0fda4744100.yaml

```
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Fixed an issue when creating a new :class:`~.AerStatevector` instance
+    from an ``numpy.ndarray`` that had non-contiguous memory. Previously,
```

```suggestion
    from a ``numpy.ndarray`` that had non-contiguous memory. Previously,
```

it'd be "a numpy.ndarray" or "an ndarray", I guess.